source/connect/favorite-connections/import-export-cli/export.txt

```diff
@@ -0,0 +1,124 @@
+.. _compass-export-connections-cli:
+
+========================================
+Export Favorite Connections with the CLI
+========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can use the |compass-short| :abbr:`CLI (Command-Line Interface)` to
+export favorite connections. This lets you share saved connections
+between workspaces and with colleagues to connect to shared deployments.
+
+About This Task
+---------------
+
+When you export favorite connections, |compass-short| exports the list
+of connections as a JSON file.
+
+By default, |compass-short| does not hide sensitive connection
+information (such as usernames and passwords) from the exported
+connection file. To learn how to protect sensitive information in
+exported connections, see :ref:`compass-encrypt-connections-cli`.
+
+Procedure
+---------
+
+To export **unencrypted** favorite connections using the |compass-short|
+:abbr:`CLI (Command-Line Interface)`, specify:
+
+- The path to the |compass| executable. The path to the executable
+  depends on your operating system.
+
+- The ``--export-connections`` flag with the value set to the
+  destination of the output file.
+
+Your operation should resemble the following prototype:
+
+.. code-block:: sh
+
+   <path-to-Compass-executable> --export-connections=<filename>
+
+Example
+-------
+
+This example exports favorite |compass-short| connections to a file with
+the path ``/tmp/compass-connections/favorites.json``. Run the following
+command in the folder containing your MongoDB Compass binary:
```

Comment:
Should this say something like, 

"On MacOS the command looks like the following:"
or 
"Each operating system is slightly different, on MacOS the command looks like the following:"